examples/waveform/higher_modes.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as pp
+from pycbc.waveform import get_td_waveform
+
+# Let's plot what our new waveform looks like
+pp.figure()
+
+# You can select sets of modes or individual modes using the 'mode_array'
+for mode_select in [None,  "22 33", "21", "22", "33", "32", "44",]: 
```

Comment:
Not sure I like the format here. I think the "proper" style for mode_array is a list, but I think we allowed weird strings to let things like config files work. Should it not be `[None, [(2,2),(3,3)] ...` here (or whatever format is expected for this after the string is parsed)?